@uportal/user-profile-menu/src/components/ProfileMenu.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="profile-menu-container" ref="userProfileMenu">
-    <button class="profile-trigger" @click="toggleMenu()">
+    <button class="profile-trigger" :class="{'active': menuOpen}" @click="toggleMenu()">
       <div class="avatar">
         <img :src="avatarUrl" v-if="avatarUrl && !useInitials" />
         <span class="fallback" v-else>{{ fallbackText }}</span>
       </div>
-      <span>{{ displayText }}</span>
+      <span v-if="useName">{{ displayText }}</span>
```

Comment:
Should this be fully removed?
Should it still be visible to a screen reader?

Reply:
Fixed